plugins/outputs/sql/README.md

```diff
@@ -155,6 +164,29 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   #  # conversion_style = "unsigned_suffix"
 ```
 
+## Schema updates
+
+The default behavior of this plugin is to create a schema for the table,
+based on the current metric (for both fields and tags). However, writing
+subsequent metrics with additional fields or tags will result in errors.
+
+If you wish the plugin to sync the column-schema for every metric,
+uncomment the `table_update_template` setting in your config file.
+
+> [!NOTE] The default setting is a generic query that your database may
+> or may not support. Consult your database's documentation for proper
+> syntax and table options.
```

Comment:
I don't see a default defined in the code, so this sentence isn't correct.

Reply:
Thanks for heads-up, indeed it was a poorly worded and confusing note. Pushed a new version, let me know if it still sounds a bit off.